spec/models/booking_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+require "spec_helper"
+
+RSpec.describe Booking, type: :model do
+  let(:user) { FactoryBot.create :user }
```

Comment:
phân biệt let vs let!

Reply:
Khai báo let sẽ được thực thi mỗi khi gối it gọi nó, khi 1 khối it gọi khai báo let 2 lần nó sẽ chỉ thực thi 1 lần và dùng cho cả khối it còn khai báo let! sẽ được thực thi trong một khối before ngầm trước khi khối it được thực thi